src/components/mypage/ReviewModal/index.tsx

```diff
@@ -0,0 +1,156 @@
+import type { ChangeEventHandler, ReactElement } from 'react'
+import { useState } from 'react'
+import { Styled } from './styled'
+import type {
+  WriteReviewModalProps,
+  ReadReviewModalProps,
+  ScoreState,
+  SCORE,
+  ReviewState,
+  CommonReviewModalProps
+} from './types'
+
+const MOCK_SCORE: SCORE = [
+  {
+    state: 'smile',
+    text: '좋아요'
+  },
+  {
+    state: 'meh',
+    text: '보통이에요'
+  },
+  {
+    state: 'sad',
+    text: '별로에요'
+  }
+]
+
+const CommonTitleContainer = ({
+  nickname,
+  productName
+}: CommonReviewModalProps): ReactElement => (
+  <Styled.TitleContainer>
+    <Styled.FirstSection>
+      <Styled.NickName>{nickname}</Styled.NickName>
+      <Styled.NormalText>님과</Styled.NormalText>
+    </Styled.FirstSection>
+    <Styled.NormalText>거래는 어땠나요?</Styled.NormalText>
+    <Styled.ProductText>{productName}</Styled.ProductText>
+  </Styled.TitleContainer>
+)
+
+const WriteReviewModal = ({
+  isOpen = true,
+  onClose,
+  onConfirm,
+  nickname = '닉네임',
+  productName = '상품이름'
+}: WriteReviewModalProps): ReactElement => {
+  const [isClickReviewIcon, setIsClickReViewIcon] = useState<boolean>(false)
+
+  const [reviewState, setReviewState] = useState<ReviewState>({
+    reviewScore: null,
+    reviewText: ''
+  })
+
+  const handleClickReviewIcon = (reviewScore: ScoreState): void => {
+    setIsClickReViewIcon(true)
+    setReviewState({ ...reviewState, reviewScore })
+  }
+
+  const handleInput: ChangeEventHandler<HTMLTextAreaElement> = e => {
+    setReviewState({
+      ...reviewState,
+      reviewText: e.target.value
+    })
+  }
+
+  return (
+    <Styled.ReviewModal isOpen={isOpen} onClose={onClose}>
+      <CommonTitleContainer
+        nickname={nickname}
+        productName={productName}></CommonTitleContainer>
+      <Styled.ReviewIconContainer>
+        {MOCK_SCORE.map(scoreItem => {
+          return (
+            <Styled.ReviewState
+              key={scoreItem.state}
+              onClick={(): void => handleClickReviewIcon(scoreItem.state)}>
+              <Styled.ReviewIcon
+                isFill={reviewState.reviewScore === scoreItem.state}
+                isGood={scoreItem.state === 'smile'}
+                type={
+                  reviewState.reviewScore === scoreItem.state
+                    ? `${scoreItem.state}Fill`
+                    : scoreItem.state
+                }></Styled.ReviewIcon>
+              <Styled.ReviewText
+                isFill={reviewState.reviewScore === scoreItem.state}>
+                {scoreItem.text}
+              </Styled.ReviewText>
+            </Styled.ReviewState>
+          )
+        })}
+      </Styled.ReviewIconContainer>
+
+      <Styled.ReviewTextArea
+        guideMessage={`${reviewState.reviewText?.length}/100`}
```

Comment:
```suggestion
        guideMessage={`${reviewState.reviewText.length}/100`}
```
기본값이 string이어서 옵셔널 체이닝 안해주어도 될 것 같습니다!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/108/commits/d45593796d2ec7c49c4161da3621af7d44fb3859)

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/108/commits/fee8074b8f9f054f36734eeda9c8b6bd81a57cc9#diff-efe33139ad392b5168600bf34f17b3e692070a8d4f1bb8f20bc10c2f0ba7ab16)